x-pack/legacy/plugins/maps/public/connected_components/map/mb/tooltip_control/tooltip_control.js

```diff
@@ -272,17 +263,41 @@ export class TooltipControl extends React.Component {
     });
   };
 
-  _syncTooltipState() {
-    if (this.props.tooltipState) {
-      this.props.mbMap.getCanvas().style.cursor = 'pointer';
-      this._showTooltip();
-    } else {
-      this.props.mbMap.getCanvas().style.cursor = '';
-      this._hideTooltip();
+  render() {
+    if (!this.props.tooltipState) {
+      return null;
     }
-  }
 
-  render() {
-    return null;
+    const tooltipAnchor = <div style={{ height: '26px', width: '26px', background: 'transparent' }}/>;
+    const isLocked = this.props.tooltipState.type === TOOLTIP_TYPE.LOCKED;
+
+    return (
+      <EuiPopover
+        id="mapTooltip"
+        button={tooltipAnchor}
+        anchorPosition="upCenter"
```

Comment:
another slight behavior change is that the anchorPosition no longer changes to leftCenter or rightCenter as the tooltip approaches the left or right edge of the map. If this is not a simple fix I would not worry about it.

Reply:
ya, not much we can do about that without lots of extra logic.